helpers/user_agent_helper.go

```diff
@@ -0,0 +1,72 @@
+package helpers
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+var userAgentPlatform = []func() string{
+	generateFirefoxUserAgent,
+	generateChromeUserAgent,
+}
+
+var firefoxVersionList = []float32{
+	52.0,
+	53.0,
+	54.0,
+	56.0,
+	57.0,
+	58.0,
+	59.0,
+	6.0,
```

Comment:
Is it normal to have a `6.0` in between `59.0` and `60.0`?

Reply:
Removed this version out 🙌